oracle/README.md

```diff
@@ -37,31 +37,57 @@ oracle/
 
 TODO: need UI for this to be relevant
 
-### Node
-In order to run the Oracle node make sure you are in the root of this project i.e `/path/to/oracle/<you are here>`
+The project can be started by executing the following steps:
 
-Copy the `.env.example` file into a new file called `.env`.  Then insert your api key into the `API_URL`
+1. Change into the `Oracle` directory
 
-**_Note:_** You do not need an api key to run any tests
+    ```bash
+    cd <path>/sway-applications/oracle/<you are here>
+    ```
 
+2. Copy and paste the `.env.example` file into a new file called `.env`
 
-Then start a local fuel-core instance
-```bash
-fuel-core run --chain packages/node/.chainConfig.json
-```
+    ```bash
+    cp packages/node/.env.example packages/node/.env
+    ```
 
-Then deploy an instance of the Oracle contract
-```bash
-forc deploy --path packages/contract --url localhost:4000 --unsigned
-```
+### Environment Variables
```

Comment:
This table breaks the flow of the entire document. It should be listed at the very bottom of this section and it should be referenced when going through the steps so that the user can see the steps in sequence followed by being able to scroll / click on the link to the relevant section rather than having a massive chunk of text in between the steps. Also, be consistent with the periods at the end of the sentences.